### PR TITLE
added more options dropdown

### PR DESCRIPTION
**Pull request by jatnerubia**
Ticket ID: [Change EDIT button, add Close and Cancel actions in View Eval Admin Screen](https://github.com/orgs/nerubia/projects/18/views/2?pane=issue&itemId=44918323)

### CODE REVIEW

File: __tests__/pages/admin/evaluation-administrations/[id]/view_evaluation.spec.ts [hunk 1]
```diff
@@ -181,7 +181,7 @@ test.describe("Admin - Evaluations", () => {
       await expect(page.getByText("Evaluation Period: 2023-01-01 to 2023-12-31")).toBeVisible()
       await expect(page.getByText("Evaluation Schedule: 2024-01-01 to 2024-01-03")).toBeVisible()
       await expect(page.getByRole("link", { name: "Progress" })).toBeVisible()
-      await expect(page.getByRole("link", { name: "Edit" })).toBeVisible()
+      await expect(page.getByRole("button", { name: "More options" })).toBeVisible()
```
Comment (emmaacha) [inline __tests__/pages/admin/evaluation-administrations/[id]/view_evaluation.spec.ts:184]: I think this is "`More actions`"?

File: src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx [hunk 2]
```diff
@@ -65,14 +84,41 @@ export const ViewEvaluationHeader = () => {
...
-              Edit
-            </LinkButton>
-            {evaluation_administration?.status === EvaluationAdministrationStatus.Draft ? (
-              <Button variant='destructive' onClick={toggleDialog}>
-                <Icon icon='Trash' />
-              </Button>
-            ) : null}
+            <Dropdown>
+              <Dropdown.Trigger>
+                <Button>More options</Button>
```
[the remark this answers is not in the document]
Comment (emmaacha) [reply src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx:89]: Also, could we add an arrow icon on the right like this?
image

**Review by emmaacha — CHANGES_REQUESTED** [review]
Just a few nits

**Comment by jatnerubia** [comment]
@emmaacha done ^^

**Review by emmaacha — APPROVED** [review]
lgtm! 🚀

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**